### ISSUE TITLE
General->Bug->High

### ISSUE THREAD

**Issue by ZhaihuanDai** [raynaglieri/MILE4STEM_FSU#42]
Avatars for TA users appeared only as cloud after logged in. Only ta 1-4 showed up. The rest would not show avatar and received notifications saying cannot find default shape/body parts in data base.
![capture](https://user-images.githubusercontent.com/36747458/38004140-ec29cbf4-3208-11e8-87df-9772c640648a.JPG)

**Comment by raynaglieri** [comment]
Something with each avatar's default appearance settings seems to have been corrupted. I was able to to make some avatars appear if I "replaced" their default body part with another one in the inventory. This may be a complex server issue that I need to discuss with Dr. Yuan.  I've attached a screen shot of the button I clicked to modify the appearance. 
<img width="1440" alt="screen shot 2018-03-27 at 10 56 15 pm" src="https://user-images.githubusercontent.com/30050477/38006320-83ad396c-3212-11e8-977d-231d133a08f6.png">

**Comment by MariyaPa** [comment]
Ok, I also checked TAs  1-20 on my end. I checked them twice, from home and from work. Here is what I found: https://drive.google.com/open?id=132wkKESYUhvbMxeCDFmeRYGgMw2ipS6f
Those that are marked "in development" are the red cloud. There is also an error associated with it (see below). 
Fengfeng thinks it's a server issue.
![red cloud problem](https://user-images.githubusercontent.com/36109730/38046586-0501d484-328e-11e8-8222-c8c51042a934.PNG)
